internal/indexmanagement/scripts.go

```diff
@@ -1,5 +1,115 @@
 package indexmanagement
 
+const indexManagementClient = `
+#!/bin/python
+
+import os, sys
+import json
+from elasticsearch import Elasticsearch
+from ssl import create_default_context
+
+def getEsClient():
+  esService = os.environ['ES_SERVICE']
+  connectTimeout = os.getenv('CONNECT_TIMEOUT', 30)
+
+  tokenFile = open('/var/run/secrets/kubernetes.io/serviceaccount/token', 'r')
```

Comment:
`try...except` missing for `open` and `read`

Reply:
@igor-karpukhin I had intentionally left that. Wherever we are calling this function, that is encapsulated between `try...except` hence any error would be caught and returned back.